web/themes/custom/nysenate_theme/nysenate_theme.theme

```diff
@@ -2912,6 +2912,28 @@ function get_issue_card($term) {
   return $issue_card;
 }
 
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function nysenate_theme_preprocess_field__node__field_block(&$variables) {
+  // Remove microsite_hero and senator_microsite_menu blocks from field
+  // rendering, as these blocks are rendered directly through
+  // page--node--microsite-page.html.twig
+  foreach ($variables['items'] ?? [] as $key => $item) {
```

Comment:
Use parenthesis around every instance of null coalesce or ternary operators.  PHP's OOO regarding these two is a little vague, and it will help with general readability.